### PR TITLE
Fix Spanish translation for "In this section"

### PR DESCRIPTION
**Pull request by chosak**
The Spanish translation for "In this section" in the secondary navigation wasn't working properly, for example on mobile on
https://www.consumerfinance.gov/es/coronavirus/asistencia-hipotecas-y-viviendas/asistencia-propietarios-de-vivienda/.

This has to do with the way the template invoked the translation code, which violated an (admittedly tricky) best practice documented at https://cfpb.github.io/consumerfinance.gov/translation/#do-not-include-the-_-call-in-a-larger-jinja2-template-data-structure.

With this change, this phrase is now properly translated.

Closes internal platform#3709 (again).

## Screenshots

|Before|After|
|-|-|
|image|image|

## Checklist

- [x] PR has an informative and human-readable title
  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)

### CODE REVIEW

File: cfgov/locale/es/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -1695,9 +1699,6 @@ msgstr "Jubilado"
 #~ msgid " of "
 #~ msgstr " de "
 
-#~ msgid "In this section"
-#~ msgstr "En esta sección"
```
Comment (anselmbradford) [inline cfgov/locale/es/LC_MESSAGES/django.po:1699]: Weird. We had this in prior, but disabled it??

Comment (anselmbradford) [reply cfgov/locale/es/LC_MESSAGES/django.po:1699]: Oh wait, I see your post comment. It was in there, but not working right.

File: cfgov/jinja2/v1/_includes/organisms/secondary-navigation.html [hunk 2]
```diff
@@ -15,8 +15,9 @@
 <nav class="o-secondary-navigation
             {{ '' if has_children else 'o-secondary-navigation__no-children' }}"
      aria-label="{{ _('Section navigation') }}">
+    {% set sec_nav_settings_label = _("In this section") %}
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/organisms/secondary-navigation.html:18]: If it matters, we had single quotes before and this is double quotes. Not sure what we usually use for these.

**Review by anselmbradford — APPROVED** [review]

File: cfgov/jinja2/v1/_includes/organisms/secondary-navigation.html (hunk 2 above)
Comment (chosak) [reply cfgov/jinja2/v1/_includes/organisms/secondary-navigation.html:18]: I don't know that it matters; I copied [the example we have documented](https://cfpb.github.io/consumerfinance.gov/translation/#do-not-include-the-_-call-in-a-larger-jinja2-template-data-structure).

File: cfgov/locale/es/LC_MESSAGES/django.po (hunk 1 above)
Comment (chosak) [reply cfgov/locale/es/LC_MESSAGES/django.po:1699]: Right. According to [this comment](https://github.com/cfpb/consumerfinance.gov/pull/5956#issuecomment-682835663) back when `validation-translations` went in, "[r]emoving all translations of a string from code" should cause that job to fail, but unfortunately this change happened before we had that check.

**Pull request merged**